### PR TITLE
Feature/kak/analysis job list view#94

### PR DESCRIPTION
**Pull request by flibbertigibbet**
Adds a list view for analysis jobs. Use it as the root view, redirected to after login. Based off of the existing `BoundaryUpload` views, which are deleted here (we will add neighborhood views instead).

Connects #94. Does not add links for downloading the logs or result files, as the API components aren't ready yet to deliver those.

image

### CODE REVIEW

File: src/angularjs/src/app/components/analysis-jobs/analysis-jobs-status.filter.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * @description
+ * Transforms analysis job status into user friendly string
+ */
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function displayStatus() {
+        var statuses = {
+            CREATED: 'Created',
```
Comment (CloudNiner) [inline src/angularjs/src/app/components/analysis-jobs/analysis-jobs-status.filter.js]: These are defined both here and in analysis-jobs.service.js. Consider defining in a constant service and reusing them here.

Comment (flibbertigibbet) [inline src/angularjs/src/app/components/analysis-jobs/analysis-jobs-status.filter.js]: Yeah, I was wondering how to improve on this. Just copied the pattern from the extant 'batch upload' filters. It is slightly different than a reverse of the map in the service; the descriptive strings here are longer than those used for the drop-downs (defined in the service). Ideally, I think we should be able to get all of these dynamically from the API though.

Comment (CloudNiner) [inline src/angularjs/src/app/components/analysis-jobs/analysis-jobs-status.filter.js]: Yeah, from the API is probably best, but we don't have an endpoint for that yet. Moving to a constants file and making each item in a dict follow the form:
```
var status = {
  IMPORTING: {
    short: 'shortdesc',
    long: 'longer description'
  },
  ...
}
```
Might be a good middle ground for now. In the place where keys/values are reversed, just loop the new status dict to find the key you want, it's not like that's going to be a big performance hit. 

Or just leave as is and make a separate issue to add the endpoint and fix this up, but either way #179 added an additional `QUEUED` status that should be reflected here before merging.

**Comment by flibbertigibbet** [comment]
Added queued status and constants file for job status options.

**Pull request merged**